pkg/controller/atlasmap/webconsole.go

```diff
@@ -0,0 +1,50 @@
+package atlasmap
+
+//go:generate go run ./.packr/packr.go
+
+import (
+	"context"
+	"github.com/RHsyseng/operator-utils/pkg/utils/openshift"
+	"github.com/atlasmap/atlasmap-operator/pkg/apis/atlasmap/v1alpha1"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+)
+
+func (r *ReconcileAtlasMap) createConsoleYAMLSamples() {
+	log.Info("Loading CR YAML samples.")
+	box := packr.New("cryamlsamples", "../../../deploy/crs")
```

Comment:
Related to my earlier comment. Since there's only one example, maybe this can all be simplified to just load the single CR example YAML.